includes/class-fontawesome.php

```diff
@@ -2986,15 +2914,24 @@ private function is_gutenberg_page() {
 	/**
 	 * Internal use only, not part of this plugin's public API.
 	 *
+	 * We can't guarantee the timing of when this global hook will be set.
+	 * So if we find that it's already set, we'll invoke it. Otherwise, we'll
+	 * assign a truthy value to it to indicate that it should be invoked as
+	 * soon as the hook is ready.
+	 *
 	 * @internal
 	 * @ignore
 	 */
 	public function print_classic_editor_icon_chooser_setup_script() {
 		?>
 	<script type="text/javascript">
-		window.tinymce
-		&& window.__FontAwesomeOfficialPlugin__setupClassicEditorIconChooser
-		&& window.__FontAwesomeOfficialPlugin__setupClassicEditorIconChooser()
+		if( window.tinymce ) {
+			if( window.__FontAwesomeOfficialPlugin__setupClassicEditorIconChooser ) {
```

Comment:
Would there be any reason to do this?
```suggestion
			if( typeof window.__FontAwesomeOfficialPlugin__setupClassicEditorIconChooser === 'function' ) {
```